AnnotatoSharedLibrary/Sources/AnnotatoSharedLibrary/Models/AnnotationText.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-public final class AnnotationText: Codable {
+public final class AnnotationText: Codable, AnnotationPart {
```

Comment:
If the above comment is done, perhaps we should separate out the properties that are common to all `AnnotationPart` and the properties that are specific to the `AnnotationText`, i.e. `content` so as to make the distinction clear